### PR TITLE
feat:Adds implementation of get_pollution_forecast with test cases

### LINKED ISSUE #22: Feature: get_pollution_forecast() python implementation
- [x] Calls API.
- [x] Converts JSON to dataframe ( add code in utitls.py).
- [x] Creates a time series chart of pollutants.
- [x] Returns chart

### LINKED ISSUE #23: get_pollution_forecast() test cases for python
- [x] Check chart title name
- [x] Check chart data ( number of columns)
- [x] API key and location parameters checks
- [x] Check pivoted data used for plotting

### LINKED ISSUE #24: get_pollution_forecast() exception handling for python
- [x] If API key is a string type
- [x] If lat and long are float 
- [x] Defensive programming check if dataframe created is not empty
- [x] API response is success.
- [x] Checks latitude and longitude ranges

### PR DESCRIPTION
**Pull request by christopheralex**
- [x] Adds implementation of get_pollution_forecast
- [x] Adds exception handling 
- [x] Unit test cases for forecast function.

### CODE REVIEW

**Comment by christopheralex** [comment]
To check working of forecast function , run the following commands in CLI
```
from airpyllution.airpyllution import get_pollution_forecast
chart = get_pollution_forecast(45.0,180.0,"your_api_key")
chart.show()
```

**Comment by christopheralex** [comment]
@danfke @mel-liow Fixed all merge conflicts and implemented unit test cases with exception handling. Kindly review.

File: tests/test_airpyllution.py [hunk 1]
```diff
@@ -82,3 +85,49 @@ def test_pollution_history(mock_api_call):
...
+        mock_incorrect_params['lon_oor'],
+        mock_params['appid']) == "Enter valid longitude values (Range should be -180<Longitude<180)"
+
+    assert airpyllution.get_pollution_forecast(
+        mock_params['lat'], 
+        mock_params['lon'], 
+        mock_incorrect_params['appid']) == 'Invalid API key. Please see http://openweathermap.org/faq#error401 for more info.'
+
+    # Functionality check with mocked data
+    forecast_chart = airpyllution.get_pollution_forecast(
```
Comment (mel-liow) [inline tests/test_airpyllution.py:125]: Nice!! this looks great!

File: tests/test_airpyllution.py [hunk 2]
```diff
@@ -23,12 +23,16 @@ def json(self):
...
+        if kwargs['params']['appid'] == 'invalid_api_key':
+            return MockResponse(mock_api_invalid_key_error, 404)
+
         return MockResponse(mock_forecast_data, 200)
     
     return MockResponse({'cod': 401, 'message': 'Invalid API key. Please see http://openweathermap.org/faq#error401 for more info.' }, 404)
 
 
-@patch('requests.get', side_effect=mocked_requests_get_pollution_history)
+@patch('requests.get', side_effect=mocked_requests_get_pollution)
```
Comment (mel-liow) [inline tests/test_airpyllution.py:35]: Good shout on changing this

File: tests/test_airpyllution.py [hunk 3]
```diff
@@ -6,7 +6,7 @@
 from airpyllution.utils import *
 
 
-def mocked_requests_get_pollution_history(*args, **kwargs):
+def mocked_requests_get_pollution(*args, **kwargs):
```
Comment (mel-liow) [inline tests/test_airpyllution.py:9]: Good shout on changing this name

**Review by mel-liow — APPROVED** [review]
Awesome - this looks really good.

File: tests/test_airpyllution.py [hunk 4]
```diff
@@ -6,7 +6,7 @@
...
     class MockResponse:
         def __init__(self, json_data, status_code):
             self.json_data = json_data
```
Comment (mel-liow) [inline tests/test_airpyllution.py:12]: Would you be able to delete the print statement on line 17 - that's my bad, shouldn't have left it in there

Comment (christopheralex) [reply tests/test_airpyllution.py:12]: Done 😊

**Pull request merged**